frontend/src/pages/org/index.ts

```diff
@@ -246,7 +316,6 @@ export class Org extends LiteElement {
       <div class="text-right">
         <sl-button
           href=${`/orgs/${this.orgId}/members/add-member`}
```

Comment:
Page currently 404s due to name change.

```suggestion
          href=${`/orgs/${this.orgId}/settings/add-member`}
```